### ISSUE TITLE
DBFBLOB: Cannot use the hidden RDD parameter in DBCreate() to specify the DBFBLOB RDD

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#1464]
VO allows the use of the hidden RDD parameter in DBCreate() to create a dbv file, accompanied with an empty dbf with no fields.

In X#, DBCreate() with DBFBLOB specified as hidden RDD fails. Not sure if doing that makes any sense, but logging it as a VO incompatibility in case someone is using it this way.

Note that VO requires the use of DBFBLOB as the "main" RDD in DBUseArea(), it does not allow specifying it as a hidden RDD in this case.

```
FUNCTION Start() AS VOID
	LOCAL cFileName AS STRING
	LOCAL blob,block AS STRING
	LOCAL nAt AS DWORD
	
	cFileName := "c:\test\rdd\dbfblob"
	
	? DbCreate(cFileName, ,"DBFCDX",,,,,{"DBFBLOB"}) // fails in X#
	
	? DbUseArea(TRUE,"DBFBLOB",cFileName + ".dbv")
	blob := AsString( BLOBDirectPut(0,"some text here") )
	? blob
	? blob += "+" + AsString( BLOBDirectPut(0,"more text") )
	? blob += "+" + AsString( BLOBDirectPut(0,"end of text") )
	
	? BLOBDirectPut(0,blob)
	
	? BLOBRootLock()
	? BLOBRootPut(blob)
	? BLOBRootUnlock()
	
	DbCloseArea()
	
	? DbUseArea(TRUE,"DBFBLOB",cFileName + ".dbv")
	
	blob := BLOBRootGet()
	DO WHILE SLen(blob) != 0
		nAt := At("+" , blob)
		? nAt
		IF nAt == 0
			block := blob
			blob := ""
		ELSE
			block := Left(blob, nAt - 1)
			blob := SubStr(blob, nAt + 1)
		ENDIF
		? BLOBDirectGet(Val(block))
	END DO
	
	
	DbCloseArea()

```

**Comment by RobertvanderHulst** [comment]
Fixed in 79edbf010872336010f18b6052510a68a438ab73

**Comment by cpyrgas** [comment]
Confirmed fixed. Note that X# does not create an empty dbf with no field specifications, but I assume this is not a problem.
